networking/multiple_networks/understanding-multiple-networks.adoc

```diff
@@ -6,57 +6,67 @@ include::_attributes/common-attributes.adoc[]
-must not be shared between tenants or customers.
-
-All of the pods in the cluster still use the cluster-wide default network
-to maintain connectivity across the cluster. Every pod has an `eth0` interface
-that is attached to the cluster-wide pod network. You can view the interfaces
-for a pod by using the `oc exec -it <pod_name> \-- ip a` command. If you
-add additional network interfaces that use Multus CNI, they are named `net1`,
-`net2`, ..., `netN`.
-
-To attach additional network interfaces to a pod, you must create configurations that define how the interfaces are attached. You specify each interface by using a `NetworkAttachmentDefinition` custom resource (CR). A CNI configuration inside each of these CRs defines how that interface is created.
-
-[id="additional-networks-provided"]
+In Kubernetes, container networking is managed through networking plugins that implement the Container Network Interface (CNI). 
+
+{product-title} uses the Multus CNI plugin, which enables the chaining of multiple CNI plugins. This setup allows you to define a default pod network during cluster installation, which handles standard network traffic. Additionally, you can define one or more custom networks based on available CNI plugins and attach them to your pods, offering flexibility for specialized network configurations such as routing or switching.
+
+By creating multiple networks within a cluster, you gain greater flexibility in network design, as well as enhanced security, scalability, and workload isolation. Multiple networks allow you to configure network topologies tailored to your specific requirements, such as multi-tenancy, traffic isolation, and fine-grained connectivity control between workloads. Multiple networks can be implemented at the namespace level, and as both primary and secondary networks. For primary networks, user-defined networks (UDNs) and network attachment definitions (NADs) are available. For secondary networks, cluster administrators can use NADs or one of several CNI plugins that are available; support for secondary networks with UDN will be added in a future version of {product-title}. Additional CNIs available can be viewed under the "Additional networks in {product-title}" section.
+
+[id="primary-networks_{context}"]
+== Primary networks
+
+In {product-title}, the term _primary network_ can refer to one of two things:
+
+// The info about CUDN will be added in a separate PR now that SDN-4930 has been merged.
+* Cluster-wide primary network. This network is global to the entire cluster. When configured, all namespaces use it unless overridden. This network is managed by OVN-Kubernetes, and is configured during cluster installation. {product-title} administrators can configure UDN to serve as the entire cluster's primary network. This option is not available for NADs. For more information about configuring UDN as your cluster-wide primary network, see. . . 
```

Comment:
I often struggle with naming conventions. I'm not sure if you want to use CUDN here where you have UDN general or if we want to keep it as is. The issue is that UDN refers to a specific CR that is not cluster scoped. For me, I find these naming issues confusing. Not sure that this helps but wanted to point it out.